ssh_config.5

```diff
@@ -1665,6 +1719,33 @@ Similarly to
 with the exception of the
 .Ev TERM
 variable, the server must be prepared to accept the environment variable.
+.It Cm StdinNull
+Redirects stdin from
+.Pa /dev/null
+(actually, prevents reading from stdin).
+This must be used when
+.Nm ssh
+is run in the background.
+A common trick is to use this to run X11 programs on a remote machine.
+For example,
+.Ic ssh shadows.cs.hut.fi emacs &
```

Comment:
Should this say `ssh -n ...`?  Or `ssh -o StdinNull=yes ...`?

Otherwise, you've silently assumed the reader has put this option in `~/.ssh/config` for `Host shadows.cs.hut.fi` (but they may not wish to do so for *every* session to that host)

Reply:
Of course! Fixed.

Reply:
I took that example from the `ssh.1` manpage where "ssh -n" is described.

 I agree that it doesn't fit well here. 

I'm just leaving this out of `ssh_config.5`, the description is perfectly fine without it.